src/sql/workbench/contrib/notebook/browser/outputs/notebookMarkdown.ts

```diff
@@ -59,7 +59,7 @@ export class NotebookMarkdownRenderer {
 		let signalInnerHTML: () => void;
 		const withInnerHTML = new Promise(c => signalInnerHTML = c);
 
-		let notebookFolder = path.dirname(this._notebookURI.fsPath) + '/';
+		let notebookFolder = this._notebookURI ? path.dirname(this._notebookURI.fsPath) + '/' : '';
```

Comment:
Is always using a / appropriate here? vs os-specific one

Reply:
I haven't added that code here but since am touching the code I can update it to os-specific one. @chlafreniere do you know if we had this for a specific reason?

Reply:
No reason that I'm aware of.